testfiles/HVACTemplate-5ZonePTAC-DOAS.idf

```diff
@@ -1761,7 +1761,8 @@
     ,                        !- Design Specification Zone Air Distribution Object Name
     ,                        !- Baseboard Heating Type
     ,                        !- Baseboard Heating Availability Schedule Name
-    ;                        !- Baseboard Heating Capacity {W}
+    ,                        !- Baseboard Heating Capacity {W}
+    None;                    !- Capacity Control Method
```

Comment:
@mjwitte should I be adding this last input to the example files? If this input didn't exist I could just default to None in ExpandObjects? Not real sure of the ExpandObjects code but it looks like all inputs need to be present. It's just a thought about num required fields.

Reply:
@mjwitte never got a response here. Do I need to add this input field or should ExpandObjects handle the missing field? It looks like HVACTemplates don't require all fields so I guess I answered my own question. Take these back out.

Reply:
@rraustad If I remember correctly, at one point all fields were expected to be present but that was relaxed later on.  Just be sure to test it both ways and make sure that min-fields in the IDD matches what expandobjects expects.